portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/ToggleSwitch.path

```diff
@@ -38,6 +38,11 @@
 	<td>//label[contains(@for,'enableSocialBookmarks')]/input/../span/span</td>
 	<td></td>
 </tr>
+<tr>
+    <td>ENABLE_SPECIFIC_TOGGLE_SWITCH</td>
+    <td>//span[contains(@id,'${key_toggleSwitch}')]</td>
```

Comment:
Please use this path `//h4[contains(.,"${key_name}")]//span[contains(@id,'toggle-switch')]`